dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -1509,8 +1509,9 @@
                     <Grid x:Name="RootElement"
                         Background="{TemplateBinding Background}"
                         Width="{StaticResource InkToolbarButtonWidth}"
-                        Height="{StaticResource InkToolbarButtonHeight}">
-
+                        Height="{StaticResource InkToolbarButtonHeight}"
+                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}" >
```

Comment:
Thanks for the fix! could you please also add 
```
contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}"
```
like you can see here: https://github.com/microsoft/microsoft-ui-xaml/blob/9052972906c8a0a1b6cb5d5c61b27d6d27cd7f11/dev/ProgressBar/ProgressBar_v1.xaml#L222 so that this works downlevel as well?

Reply:
Hey Stephen, I did not add 'contract7NotPresent' because I did not find it on any other style/template in the file InkToolbar_themeresources.xaml. If it has to be added then I think we should add this to other styles/templates in this file as well.

Reply:
I see, I think that is an oversight, however we can save it for a later update.